stackable-certs/src/keys/rsa.rs

```diff
@@ -10,7 +10,8 @@ use tracing::instrument;
 
 use crate::keys::KeypairExt;
 
-pub const DEFAULT_RSA_BIT_SIZE: usize = 2048;
+pub const DEFAULT_BIT_SIZE: usize = 4096;
+pub const MINIMUM_BIT_SIZE: usize = 2048;
```

Comment:
**thought (non-blocker)**: Almost looks like an enum would suffice for the valid key lengths (if we are using the standard base2.

But that would add a bit more code with having to impl TryFrom/TryInto for usize.

Reply:
Sure, we can do that.

I don't know if we need a `TryFrom` impl, but we can definitely provide a `From<Enum> for usize` impl.

Reply:
I like that!